src/index.js

```diff
@@ -1,30 +1,37 @@
 const brs = require('brs');
 const glob = require('glob');
 const path = require('path');
+const util = require('util');
+const TapMochaReporter = require('tap-mocha-reporter');
 
-function findBrsFiles(testFile, sourceDir, cb) {
+async function findBrsFiles(sourceDir) {
     let searchDir = sourceDir || 'source';
     const pattern = path.join(searchDir, '**', '*.brs');
-    glob(pattern, (err, files) => {
-        if (!err) {
-            files.push(testFile);
-            cb(files);
-        }
-    });
+    return util.promisify(glob)(pattern);
 }
 
-async function runTest(files) {
-    let rocaBrs = path.join(__dirname, "..", "resources", "roca.brs");
+async function runTest(files, reporter) {
+    let rocaFiles = [
+        "tap.brs",
+        "roca_lib.brs",
+        "roca_main.brs"
+    ].map(basename => path.join(__dirname, "..", "resources", basename));
 
     try {
-        const result = await brs.execute([ rocaBrs, ...files ]);
+        let reporterStream = new TapMochaReporter(reporter);
+        await brs.execute([ ...rocaFiles, ...files], {
+            stdout: reporterStream,
+            stderr: process.stderr
+        });
```

Comment:
I didn't know we could do this 🤯 I was trying to figure out how to override stdout in a node process